src/main/kotlin/com/arsiu/eduhub/service/util/mail/EmailServiceImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.arsiu.eduhub.service.util.mail
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.mail.SimpleMailMessage
+import org.springframework.mail.javamail.JavaMailSender
+import org.springframework.stereotype.Service
+
+@Service
+class EmailServiceImpl(
+    private val jms: JavaMailSender,
+    @Value("\${spring.mail.username}") private val sender: String
+) : EmailService {
+
+    override fun sendMail(details: EmailParameters) {
+        val smm = SimpleMailMessage()
+        smm.from = sender
+        smm.setTo(details.receiver)
+        smm.text = details.msg
+        smm.subject = details.subject
```

Comment:
It's time to get familiar with [apply](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html) function